gcp_variant_transforms/pipeline_common.py

```diff
@@ -329,3 +330,36 @@ def generate_unique_name(job_name):
   return '-'.join([job_name,
                    datetime.now().strftime('%Y%m%d-%H%M%S'),
                    str(uuid.uuid4())])
+
+
+BQ_CREATE_PARTITIONED_TABLE_COMMAND = (
+    'bq mk --table '
+    '--range_partitioning=start_position,0,{TOTAL_BASE_PAIRS},{PARTITION_SIZE} '
+    '--clustering_fields=start_position,end_position '
+    '{FULL_TABLE_ID} {SCHEMA_FILE_PATH}')
+
+def create_output_table(full_table_id, total_base_pairs, schema_file_path):
+  # type: (str, int, str) -> None
+  """Creates an integer range partitioned table using `bq mk table...` command.
```

Comment:
no-op: Umm I only wanted a top line you've added here, I think it would had been sufficient :D .

Still doesn't hurt to have a nice write up though! Explains well why we have to jump through so many hoops.